tvb_ext_bucket/handlers.py

```diff
@@ -22,15 +22,19 @@ def get(self):
         }
```

Comment:
I think some of the files (like this one) are missing the copyright header.
Could we have that in all files produced by us ?